server/api/v2/openapi.json

```diff
@@ -1744,6 +1988,32 @@
           }
         }
       },
+      "simplifiedReport": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "rule_fqdn": {
```

Comment:
is rule_fqdn an agreed name? IIRC It's not used anywhere in our API and people always have different understanding of this name. Most people expect the error key too if it's called "fully qualified......"

Reply:
Let me run that through the right people and come back to you

Reply:
We have some inconsistencies when it comes to naming rules (with or without error key, with '.report', etc) due supporting different clients, but we kind of agreed on using `rule_fqdn|erorr_key` combination and call it `rule_selector` (e.g v2 `clusters_detail` endpoint).

The `rule_fqdn` name comes from the [rules specification](https://docs.google.com/document/d/1bGwR1J7bezb2k-KbJ5lVtEd4ftq_9WSVK6vd_3MllwY/edit#heading=h.i00d53mlijgv) itself.